main.go

```diff
@@ -343,6 +367,15 @@ func transformCompile(args []string) ([]string, error) {
 
 	blacklist := buildBlacklist(files, info, pkg)
 
+	pkgDebugDir := ""
+	if envGarbleDebugDir != "" {
+		pkgDebugDir = filepath.Join(envGarbleDebugDir, pkgPath)
```

Comment:
like I said earlier, please split `pkgPath` first.

Reply:
You need to consider that the import paths have different numbers of folders not only two as in the test.
I dont get why you want to split it first.

What makes sense is to do `filepath.FromSlash(pkgPath)` to convert the `/` to `\` for Windows support.

Reply:
What I was thinking was:

```
parts := strings.Split(pkgPath, "/")
pkgDebugDir = filepath.Join(envGarbleDebugDir, parts...)
```

But your suggestion with FromSlash is better - it sounds good to me.